ParkingZoneApp/ViewModels/ParkingZonesVMs/ListItemVM.cs

```diff
@@ -14,13 +14,44 @@ public class ListItemVM
         [Required]
         public string Address { get; set; }
 
+        public ICollection<ParkingSlot> ParkingSlots { get; set; }
+
+        public int FreeSlots
+        {
+            get => FreeSlot();
+        }
+
+        public int SlotsInUse
+        {
+            get => SlotInUse();
+        }
+
         public ListItemVM() { }
 
         public ListItemVM(ParkingZone parkingZone)
         {
             Id = parkingZone.Id;
             Name = parkingZone.Name;
             Address = parkingZone.Address;
+            ParkingSlots = parkingZone.ParkingSlots;
+        }
+        private int SlotInUse()
```

Comment:
how about naming like this which is more informative
```suggestion
        private int CountSlotInUse()
```